dbld/images/centos7/Dockerfile

```diff
@@ -3,11 +3,13 @@ MAINTAINER Andras Mitzki <andras.mitzki@balabit.com>
 
 RUN yum install -y wget epel-release
 
-RUN cd /etc/yum.repos.d && wget https://copr.fedorainfracloud.org/coprs/czanik/syslog-ng39/repo/epel-7/czanik-syslog-ng39-epel-7.repo
+RUN cd /etc/yum.repos.d && wget https://copr.fedorainfracloud.org/coprs/czanik/syslog-ng311/repo/epel-7/czanik-syslog-ng311-epel-7.repo
 
 ADD dev-dependencies.txt .
 RUN cat dev-dependencies.txt | grep -v "#" | xargs yum -y  install
-
+ENV GRADLE_HOME=/opt/gradle
```

Comment:
Thanks for the note. After rethinking it I also agreed with you (it seems a packaging issue).
@czanik could you fix the gradle package?
The mentioned alternative solution is ok, but it comes with additional maintenance work: we need to update every time to latest package version e.g: 4.0.1 - 4.0.2 ...